src/themes/default/components/core/ProductGalleryCarousel.vue

```diff
@@ -16,41 +16,12 @@
         v-for="(images, index) in gallery"
         :key="images.src">
         <div class="product-image-container bg-cl-secondary" :class="{'video-container w-100 h-100 flex relative': images.video}">
-          <img
-            v-show="placeholderImagesMap[index]"
-            key="placeholderImage"
-            class="inline-flex mw-100"
-            src="/assets/placeholder.svg"
-            ref="images"
-            :alt="productName | htmlDecode"
-          >
-          <img
-            v-if="!lowerQualityImagesErrorsMap[index] || isOnline"
-            v-show="lowerQualityImagesMap[index]"
-            key="lowQualityImage"
+          <product-image
             class="product-image inline-flex mw-100"
-            :src="images.loading"
-            @load="lowerQualityImageLoaded(index, true)"
-            @error="lowerQualityImageLoaded(index, false)"
-            ref="images"
-            :alt="productName | htmlDecode"
-            data-testid="productGalleryImage"
-            itemprop="image"
-          >
-          <img
-            v-if="!highQualityImagesErrorsMap[index] || isOnline"
-            v-show="highQualityImagesLoadedMap[index]"
-            key="highQualityImage"
-            :src="images.src"
-            @load="highQualityImageLoaded(index, true)"
-            @error="highQualityImageLoaded(index, false)"
-            class="product-image inline-flex pointer mw-100"
-            ref="images"
-            @dblclick="openOverlay"
+            :images="images"
+            :hidden="hideImageAtIndex === index"
```

Comment:
lets use v-show instead of hidden, because hiding these images is only for carousel purpose